supriya/scsynth.py

```diff
@@ -72,73 +79,110 @@ def __post_init__(self):
         ):
             raise ValueError("Insufficient audio buses")
 
+    def _find_executable_path(self, executable: Optional[str] = None):
```

Comment:
Why move this onto the Options class?

The tell is that...
- the method doesn't refer to `self`
- the `find()` function you reimplemented has to instantiate an entire Options object just to find the executable
- it's treated as a private method, so the end-user doesn't care where it's implemented

Reply:
It is a bit contrived, I agree.  I wanted to restore backward compatibility first.  But after all these changes, `find()` is not used anywhere in the code, and I think it is not needed anymore since the user will always instantiate at least one copy of `Options()`.  So the function can be eventually deprecated.

Reply:
My gut feeling is that the function is useful outside the context of an Options class (e.g. an application might check for the presence of a server executable _before_ booting), and it's weird to need to instantiate a class to use what's effectively a static method.

Reply:
Right.  I forgot non-trivial options are given to the Server class only with `boot()` or `connect()`.